document-generator-core/src/main/java/uk/gov/companieshouse/document/generator/core/document/models/DescriptionValues.java

```diff
@@ -0,0 +1,197 @@
+/**
+ * Autogenerated by Avro
+ *
+ * DO NOT EDIT DIRECTLY
+ */
+package uk.gov.companieshouse.document.generator.core.document.models;
+
+import org.apache.avro.specific.SpecificData;
+
+@SuppressWarnings("all")
+@org.apache.avro.specific.AvroGenerated
+public class DescriptionValues extends org.apache.avro.specific.SpecificRecordBase implements org.apache.avro.specific.SpecificRecord {
+  private static final long serialVersionUID = -6491296529232799506L;
+  public static final org.apache.avro.Schema SCHEMA$ = new org.apache.avro.Schema.Parser().parse("{\"type\":\"record\",\"name\":\"DescriptionValues\",\"namespace\":\"uk.gov.companieshouse.document.generator.core.document.models\",\"fields\":[{\"name\":\"date\",\"type\":{\"type\":\"string\",\"avro.java.string\":\"String\"}}]}");
+  public static org.apache.avro.Schema getClassSchema() { return SCHEMA$; }
+  @Deprecated public java.lang.String date;
+
+  /**
+   * Default constructor.  Note that this does not initialize fields
+   * to their default values from the schema.  If that is desired then
+   * one should use <code>newBuilder()</code>.
+   */
+  public DescriptionValues() {}
+
+  /**
+   * All-args constructor.
+   * @param date The new value for date
+   */
+  public DescriptionValues(java.lang.String date) {
+    this.date = date;
+  }
+
+  public org.apache.avro.Schema getSchema() { return SCHEMA$; }
+  // Used by DatumWriter.  Applications should not call.
+  public java.lang.Object get(int field$) {
```

Comment:
I would try to be consistent if you are using javadocs, try to apply to all methods.